config/auth.json

```diff
@@ -1,3 +1,3 @@
 {
- "access_token":""
+ "access_token":"xoxp-55672736423-55668248624-59266744898-5233d12dc8"
```

Comment:
Why are you including an access-token in the committed code? This is almost certainly a bad idea.